src/query/tsdb/remote/client.go

```diff
@@ -136,15 +136,15 @@ func (c *grpcClient) fetchRaw(
 	ctx context.Context,
 	query *storage.FetchQuery,
 	options *storage.FetchOptions,
-) (encoding.SeriesIterators, error) {
+) (encoding.SeriesIterators, bool, error) {
```

Comment:
Maybe `fetchRawResult struct` here too?  Just to avoid all the extra return args and updates required if we introduce another "metadata" like field.